cmake build on Windows is failing

I don't speak cmake very well, but I think this is supposed to work for me on Windows (I have VS 2019 installed):

```
PS C:\Users\Arnoud\Documents\GitHub\codeparser\build> cmake ..
-- Building for: Visual Studio 16 2019
-- Selecting Windows SDK version 10.0.17763.0 to target Windows 10.0.18363.
-- The C compiler identification is MSVC 19.21.27702.2
-- The CXX compiler identification is MSVC 19.21.27702.2
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.21.27702/bin/Hostx64/x64/cl.exe
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.21.27702/bin/Hostx64/x64/cl.exe - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.21.27702/bin/Hostx64/x64/cl.exe
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.21.27702/bin/Hostx64/x64/cl.exe - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- PACLET: CodeParser
-- WOLFRAMKERNEL: C:/Program;Files/Wolfram;Research/Mathematica/12.0/wolfram.exe
-- BUILD_DOCS: OFF
-- BUILD_EXE: OFF
-- BUILD_EXPR_LIB: ON
-- USE_MATHLINK: ON
-- NISSUES: OFF
-- NABORT: OFF
-- LOCAL_BUILD: OFF
-- VERSION_NUMBER: 1200
-- SYSTEMID: Windows-x86-64
-- PACLET_VERSION: 1.1
-- VERSION_NUMBER is below 1210; disabling BUILD_EXPR_LIB
-- WOLFRAMLIBRARY_INCLUDE_DIR: C:/Program;Files/Wolfram;Research/Mathematica/12.0/SystemFiles/IncludeFiles/C
-- MATHLINK_INCLUDE_DIR: C:/Program;Files/Wolfram;Research/Mathematica/12.0/SystemFiles/Links/MathLink/DeveloperKit/Windows-x86-64/CompilerAdditions
-- MATHLINK_LIB_DIR: C:/Program;Files/Wolfram;Research/Mathematica/12.0/SystemFiles/Links/MathLink/DeveloperKit/Windows-x86-64/CompilerAdditions
-- MATHLINK_LIB: MATHLINK_LIB-NOTFOUND
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
MATHLINK_LIB
    linked by target "codeparser-lib" in directory C:/Users/Arnoud/Documents/GitHub/codeparser

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
PS C:\Users\Arnoud\Documents\GitHub\codeparser\build>
```

--- a/HowToBuild.md
+++ b/HowToBuild.md
@@ -7,6 +7,7 @@ CodeParser uses C++11 features and requires a compiler that can support at least
 CodeParser uses CMake to generate build scripts.
 
 Here is an example transcript using the default make generator to build CodeParser:
+
 ```
 cd codeparser
 mkdir build
@@ -14,10 +15,19 @@ cd build
 cmake ..
 cmake --build . --target paclet
 ```
+
 The result is a directory named `paclet` that contains the WL package source code and a built CodeParser `.paclet` file for installing.
 
 Specify `MATHEMATICA_INSTALL_DIR` if you have Mathematica installed in a non-default location:
+
 ```
 cmake -DMATHEMATICA_INSTALL_DIR=/Applications/Mathematica111.app/Contents/ ..
+cmake --build . --target paclet
+```
+
+On Windows:
+
+```
+cmake -DMATHEMATICA_INSTALL_DIR="C:/Program Files/Wolfram Research/Mathematica/12.0" ..
 cmake --build . --target paclet
 ```

--- a/cmake/WolframKernel.cmake
+++ b/cmake/WolframKernel.cmake
@@ -1,7 +1,7 @@
 
 if(NOT DEFINED MATHEMATICA_INSTALL_DIR)
 if(CMAKE_HOST_WIN32 OR CYGWIN)
-	set(MATHEMATICA_INSTALL_DIR C:/Program Files/Wolfram Research/Mathematica/12.0)
+	set(MATHEMATICA_INSTALL_DIR "C:/Program Files/Wolfram Research/Mathematica/12.0")
 elseif(CMAKE_HOST_APPLE)
 	set(MATHEMATICA_INSTALL_DIR /Applications/Mathematica.app/Contents)
 else()

